services/galley/src/Galley/API/MLS/Message.hs

```diff
@@ -776,17 +778,17 @@ processCommitWithAction ::
   Maybe ClientId ->
   Maybe ConnId ->
   Local Data.Conversation ->
+  ConversationMLSData ->
   ClientMap ->
   Epoch ->
-  GroupId ->
   ProposalAction ->
   Sender 'MLSPlainText ->
   Commit ->
   Sem r [LocalConversationUpdate]
-processCommitWithAction qusr senderClient con lconv cm epoch groupId action sender commit =
+processCommitWithAction qusr senderClient con lconv mlsMeta cm epoch action sender commit =
```

Comment:
Can you check if `ConvNotFound` can be removed from this one too?

Reply:
Unfortunately it can't because of downstream `checkGroup`.